ui/src/widgets/ui-text/UIText.vue

```diff
@@ -22,7 +23,8 @@ export default {
     computed: {
         ...mapState('data', ['messages', 'properties']),
         value: function () {
-            return this.messages[this.id]?.payload
+            console.log(this.messages[this.id]?.payload || 'No Message Received')
+            return this.messages[this.id]?.payload || 'No Message Received'
```

Comment:
This would prevent empty string and `false` and `null` being displayed and would show 'No Message Received' instead.  Is that deliberate?

Perhaps this should be if there is no payload return empty string?

```suggestion
            const m = this.messages[this.id] || {}
            if (Object.prototype.hasOwnProperty.call(m, 'payload')) {
                return m.payload
            }
            return ''
```